server/sonar-server/src/test/java/org/sonar/server/organization/DefaultOrganizationProviderRule.java

```diff
@@ -19,12 +19,28 @@
  */
 package org.sonar.server.organization;
 
-import org.junit.rules.ExternalResource;
+import java.util.Optional;
+import org.sonar.db.DbSession;
 import org.sonar.db.DbTester;
+import org.sonar.db.organization.OrganizationDto;
+import org.sonar.server.property.InternalProperties;
 
-public class DefaultOrganizationProviderRule extends ExternalResource implements DefaultOrganizationProvider {
+import static com.google.common.base.Preconditions.checkState;
+
+public class DefaultOrganizationProviderRule implements DefaultOrganizationProvider {
 
   private final DbTester dbTester;
+  private OrganizationDto dto;
+
+  private static DefaultOrganization toDefaultOrganization(OrganizationDto organizationDto) {
```

Comment:
usually, private code is after public code that uses it
I thought for a second this was a static utility method